packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -141,6 +151,8 @@ class TestCommand extends FlutterCommand {
     }
 
     commandValidator();
+    final List<String> names = argResults['name'];
```

Comment:
How is that possible?  Won't they get passed through as `null` to `runTests()`, thus overriding the default values and causing us to try to run `for (String name in null)`?